documentation/src/main/asciidoc/user_guide/streams.adoc

```diff
@@ -534,5 +539,41 @@ public class WordFrequencyExample {
 
 This way you can still utilize all of the cores locally when calculating the most frequent element.
 
+*Remove specific entries*
+
+Distributed streams can also be used as a way to modify data where it lives.
+For example you may want to remove all entries in your cache that contain
+a specific word.
+
+[source,java]
+----
+public class RemoveBadWords {
+   public static void main(String[] args) {
+      // Lines removed
+      String word = ..
+
+      c1.entrySet().parallelStream()
+         .filter(e -> e.getValue().contains(word))
+         .forEach((c, e) -> c.remove(e.getKey());
+----
+
+If we carefully note what is serialized and what is not, we notice that only the word along
+with the operations are serialized across to other nods as it is captured by the lambda.
+However the real saving piece is that the cache operation is performed on the primary
+owner thus reducing the amount of network traffic required to remove these values from the
+cache. The cache is not captured by the lambda as we provide a special BiConsumer method
+override that when invoked on each node passes the cache to the BiConsumer 
```

Comment:
How about "The cache itself is not serializable, so it cannot be captured by a serializable lambda. Instead, we have to use the `forEach(BiConsumer<Cache<K, V>, ? super V> action)` overload, so that the lambda receives the cache as a parameter when it is executed on each node.`

Also, missing `.` at the end.